mitmproxy/tools/cmdline.py

```diff
@@ -468,20 +474,15 @@ def proxy_options(parser):
             requests. Format: username:password
         """
     )
+
     rawtcp = group.add_mutually_exclusive_group()
     rawtcp.add_argument("--raw-tcp", action="store_true", dest="rawtcp")
     rawtcp.add_argument("--no-raw-tcp", action="store_false", dest="rawtcp",
                         help="Explicitly enable/disable experimental raw tcp support. "
                         "Disabled by default. "
                         "Default value will change in a future version."
                         )
-    websockets = group.add_mutually_exclusive_group()
```

Comment:
As long as we don't have any UI indication in mitmproxy (not even in the event log), we should keep the current disabled-by-default behaviour. I'd love to have the TCP mode enabled by default as well, but I think we really can't justify it as long as a user is unable to see what's going on.

In order to get this merged quickly, let's revert the changes here and then follow up with another PR that provides a very simple UI integration and enables it by default.